doc/releases/changelog-dev.md

```diff
@@ -305,6 +305,11 @@
 
 <h3>Breaking changes</h3>
 
+* The tape method `get_operation` can also now return the operation index in the tape, and it can be
+  activated by setting the `return_op_index` to `True`: `get_operation(idx, return_op_index=True)`. It will become
+  the default in the future.
```

Comment:
When? Always a good idea to have a timeline for these things so we know when we need to go back and complete the deprecation cycle.

Reply:
0.29 introduced this change, 0.30 we switch